tests/kokkos/kokkos_utils_tests.cpp

```diff
@@ -288,9 +288,15 @@ void test_view_reduction(const Scalar a=Scalar(0.0), const int begin=0, const in
   Kokkos::View<Scalar*> results ("results", 1);
   const auto results_h = Kokkos::create_mirror_view(results);
 
+  // Set team size
+  int team_size = ExeSpace::concurrency();
+  if (ekat::OnGpu<ExeSpace>::value) {
+    team_size /= (ekat::is_single_precision<Real>::value ? 64 : 32);
+  }
+
   // parallel_for over 1 team, i.e. call view_reduction once
   const auto policy =
-    ekat::ExeSpaceUtils<ExeSpace>::get_default_team_policy(1, view_length);
+    ekat::ExeSpaceUtils<ExeSpace>::get_team_policy_force_team_size(1, team_size);
```

Comment:
It might not matter here whether the vector length is 1 or larger. But I _think_ we should give the possibility in `get_team_policy` to specify the vector length (ignored on non-GPU builds).